tools/docker/Dockerfile

```
@@ -95,6 +95,10 @@ COPY --link --from=build --chown=root:root /usr/local/renovate/ /usr/local/renov
 # make our node binary available as last in path
 RUN ln -sf /usr/local/renovate/node /bin/node
 
+# ensure default base and cache directories exist.
+RUN mkdir -p /tmp/renovate/cache && \
+    chmod -R 775 /tmp/renovate/cache
```

```suggestion
RUN mkdir -p /tmp/renovate/cache && \
    chmod -R 775 /tmp/renovate/
```

actually this is the correct command, -R changes the content of the dir recursively and not the parents.

---

I prefer recursive actually, but don't mind too much